app/view_objects/results_view.rb

```diff
@@ -398,4 +422,27 @@ def radii_for_suggestions
     radius_for_all_england = nil
     [10, 20, 50].reject { |rad| rad <= radius.to_i } << radius_for_all_england
```

Comment:
Feels like this method might be getting some scope creep. Would be nice if we could seperate generating queries generally from generating them from the parameters (via methods such as `with_salary?` etc)